actions/delegator/setup-token/src/main.ts

```diff
@@ -81,7 +73,6 @@ async function run(): Promise<void> {
         run_id: process.env.GITHUB_RUN_ID,
         run_number: process.env.GITHUB_RUN_NUMBER,
         sha: process.env.GITHUB_SHA,
-        triggering_actor_id: current_run.triggering_actor.id,
```

Comment:
is it the same as actor_id?

Reply:
Most often it's the same, but it could be different. For example, the original actor_id could be you, but if I trigger a re-run of the job you originally triggered, then the triggering_actor_id could be my id for the re-run while actor_id is still you.